desktop/cmp/input/CodeInput.js

```diff
@@ -215,11 +223,13 @@ export class CodeInput extends HoistInput {
             editor.setSize(width, height);
         }
 
+        const val = this.tryPrettyPrint(editor.getValue());
+        editor.setValue(val);
         return editor;
     }
 
     createDefaults() {
-        const {disabled, mode, linter, autoFocus} = this.props;
+        const {disabled, mode, linter, autoFocus} = this.props;  // TODO: None of these will get reset on prop change
```

Comment:
This PR contains the fix to ensure that disabled can be toggled. Should we do the same for these other options or are assuming that the other three are truly more like immutable configs on the input.